doc/src/manual/types.md

```diff
@@ -624,7 +621,7 @@ arguments to the object constructor.
 Since the type `Point{Float64}` is a concrete type equivalent to `Point` declared with [`Float64`](@ref)
```

Comment:
there's some nonstandard indentation in the two `norm` examples in the collapsed block here, I'll get them at some point if you don't